drivers/pic.c

```diff
@@ -52,3 +52,10 @@ void init_pic(void) {
     outb(PIC1_PORT_DATA, 0xFF);
     outb(PIC2_PORT_DATA, 0xFF);
 }
+
+void pic_enable_irq(uint8_t pic, uint8_t irq) {
```

Comment:
I would define enum pic_device_sel { PIC1_DEVICE_SEL, PIC2_DEVICE_SEL }; typedef enum pic_device_sel pic_device_sel_t; and use instead of magic numbers.